wazuh/recipes/wazuh_api.rb

```diff
@@ -26,7 +26,10 @@
 package ['nodejs', 'wazuh-api']
 
 
-api_keys = Chef::EncryptedDataBagItem.load('wazuh_secrets', 'api')
+api_keys = Chef::EncryptedDataBagItem.load(
+  node['wazuh']['api']['data_bag']['name'],
+  node['wazuh']['api']['data_bag']['item']
+)
```

Comment:
👍 for the ability to override data bag settings

Can you replace `Chef::EncryptedDataBagItem.load(...)` with `data_bag_item(...)`?  Docs for that are [here](https://docs.chef.io/data_bags.html)

> If `item` is encrypted, `data_bag_item` will automatically decrypt it using the key specified above, or (if none is specified) by the `Chef::Config[:encrypted_data_bag_secret]` method, which defaults to `/etc/chef/encrypted_data_bag_secret`.